### PR TITLE
revert: downgrade Ignite to v0.4.4

### PR DESCRIPTION
**Pull request by ydcjeff**
### Description

revert: downgrade Ignite to v0.4.4

ref: a0e9ce446a901bda0b481ce4c5ce329654bf3c9d

### Additional context

---

### What is the purpose of this pull request?

- [ ] Bug fix
- [ ] New feature
- [x] Other

### CODE REVIEW

File: src/templates/template-vision-segmentation/utils.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from ignite.handlers.early_stopping import EarlyStopping
 from ignite.handlers.terminate_on_nan import TerminateOnNan
 from ignite.handlers.time_limit import TimeLimit
+from ignite.handlers.timing import Timer
```
Comment (ydcjeff) [inline src/templates/template-vision-segmentation/utils.py:18]:
```suggestion
```

File: src/templates/template-vision-dcgan/utils.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 from ignite.handlers.early_stopping import EarlyStopping
 from ignite.handlers.terminate_on_nan import TerminateOnNan
 from ignite.handlers.time_limit import TimeLimit
+from ignite.handlers.timing import Timer
```
Comment (ydcjeff) [inline src/templates/template-vision-dcgan/utils.py:18]:
```suggestion
```

File: src/templates/template-vision-classification/utils.py [hunk 3]
```diff
@@ -15,6 +15,7 @@
 from ignite.handlers.early_stopping import EarlyStopping
 from ignite.handlers.terminate_on_nan import TerminateOnNan
 from ignite.handlers.time_limit import TimeLimit
+from ignite.handlers.timing import Timer
```
Comment (ydcjeff) [inline src/templates/template-vision-classification/utils.py:18]:
```suggestion
```

File: src/templates/template-text-classification/utils.py [hunk 4]
```diff
@@ -15,6 +15,7 @@
 from ignite.handlers.early_stopping import EarlyStopping
 from ignite.handlers.terminate_on_nan import TerminateOnNan
 from ignite.handlers.time_limit import TimeLimit
+from ignite.handlers.timing import Timer
```
Comment (ydcjeff) [inline src/templates/template-text-classification/utils.py:18]:
```suggestion
```

File: src/templates/template-common/utils.py [hunk 5]
```diff
@@ -15,6 +15,7 @@
 from ignite.handlers.early_stopping import EarlyStopping
 from ignite.handlers.terminate_on_nan import TerminateOnNan
 from ignite.handlers.time_limit import TimeLimit
+from ignite.handlers.timing import Timer
```
Comment (ydcjeff) [inline src/templates/template-common/utils.py:18]:
```suggestion
```

**Comment by trsvchn** [comment]
This commit "Reverts  downgrading to 0.4.4" or it "Downgrades to 0.4.4"? A little bit confusing

**Comment by trsvchn** [comment]
as far as I remember we decided to stick with nightly version until the 0.4.5 release with switching to 0.4.5 after it's release
am I right? @vfdev-5

**Comment by ydcjeff** [comment]
> as far as I remember we decided to stick with nightly version until the 0.4.5 release with switching to 0.4.5 after it's release
> am I right? @vfdev-5

Yes, we decided to stick with nightly version. But adding `--pre` in requirements.txt doesn't install nightly version. Adding `--pre` to install commands install nightly version of other requirements.

**Pull request merged**